deploy/1-deploy-oracles.ts

```diff
@@ -11,13 +11,17 @@ const ADDRESSES = {
     VAIAddress: testnetDeployments.Contracts.VAI,
     pythOracleAddress: "0xd7308b14BF4008e7C7196eC35610B1427C5702EA",
     sidRegistryAddress: "0xfFB52185b56603e0fd71De9de4F6f902f05EEA23",
+    acm: undefined,
```

Comment:
The ACM in testnet is https://testnet.bscscan.com/address/0x45f8a08f534f34a97187626e05d4b6648eeaa9aa

Reply:
This should be available from the `testnetDeployments` config